ocs_ci/helpers/dr_helpers_ui.py

```
@@ -1838,3 +1838,879 @@ def navigate_using_fleet_virtualization(acm_obj):
+            log.warning(f"Could not get last sync time text: {e}")
+    else:
+        log.error("Last sync time NOT found in modal")
+        verification_results.append(("Last sync time present", False))
+
+    acm_obj.take_screenshot(f"manage_dr_modal_verified_{app_name}")
+
+    # Close the modal
+    try:
+        acm_obj.do_click(acm_loc["manage-dr-modal-close"], timeout=10)
+        log.info("Closed Manage DR modal")
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)